crates/diesel_models/src/payment_intent.rs

```diff
@@ -39,8 +34,6 @@ pub struct PaymentIntent {
     pub off_session: Option<bool>,
     pub client_secret: Option<String>,
     pub active_attempt_id: String,
-    pub business_country: Option<storage_enums::CountryAlpha2>,
-    pub business_label: Option<String>,
     #[diesel(deserialize_as = super::OptionalDieselArray<pii::SecretSerdeValue>)]
     pub order_details: Option<Vec<pii::SecretSerdeValue>>,
```

Comment:
- Order details can be type instead of a SecretSerdeValue
- connector_metadata as SecretSerdeValue
- feature_metadata as SecretSerdeValue
- allowed_payment_method_types as SecretSerdeValue
- introduce a field called merchant_profile_id and make it mandatory
- introduce a field called payment_link_enabled
- introduce a field called payment_link_config and type as PaymentCreatePaymentLinkConfig

Reply:
what is `merchant_profile_id` we already have a field called `profile_id`

Reply:
> introduce a field called payment_link_enabled

Instead of this field being called as `payment_link_enabled`, how about `enable_payment_link` since this captures the intent of the merchant.